docs/dictionary/command/visual-effect.lcdoc

```diff
@@ -15,21 +15,21 @@ OS: mac, windows, linux, ios, android, html5
 Platforms: desktop, mobile
 
 Example:
-visual effect dissolve
+visual effect "dissolve"
 go next card
 
 Example:
 -- This handler in an object will cause to hide then show 
 --  with a visual effect when clicked.
 on mouseUp
   hide me
-  show me with visual effect iris close slow
+  show me with visual effect "iris" close slow
```

Comment:
Hi Sam,
OT, Not to do with the adding of quotes here, which is a good idea.
Can you look at the entries for 'Hide' and 'Show' commands? In 'hide' there is a comment added regarding QT effects. Do you think this should be added to both the 'show' entry as well as this one for 'Visual-effect'? Perhaps also improving the disambiguation around QT not being used and that visual effects are or are not usable with 'hide' and/or 'show'. It's not entirely clear at the moment.
Thanks, Sean.